cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -22,6 +22,7 @@ pub mod args;
 mod emit;
 #[cfg(test)]
 mod emit_tests;
+pub(crate) mod encoding;
```

Comment:
Not sure about what visibility level is best for this module...

Reply:
I think crate-public is good for now. We should probably reconsider some of the other `pub` submodules, to be honest; IMHO we expose too much implementation already; but that's out-of-scope here :-)